jar-infer/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id "net.ltgt.errorprone" version "0.0.13"
+    id "java"
+    id "application"
+}
+// we use this config to get the path of the JDK 9 javac jar, to
+// stick it in the bootclasspath when running tests
+configurations.maybeCreate("epJavac")
+
+sourceCompatibility = 1.8
+version = '0.1'
+jar {
+    manifest {
+        attributes 'Implementation-Title': 'WALA Start',
+                   'Implementation-Version': version
+    }
+}
+
+repositories {
+    mavenCentral()
+    // uncomment if you want to use wala.dalvik or wala.scandroid
+    //maven { url "https://raw.github.com/msridhar/maven-jars/mvn-repo/" }
+}
+
+dependencies {
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.shrike', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.util', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.core', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.js', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.js.rhino', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.java', version: '1.4.3'
+    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.java.ecj', version: '1.4.3'
```

Comment:
We eventually want to move this to `deps` too, but is fine here for now, I think.